Project/core/src/main/java/View/screen/GameScreenComponents/TurnBar.java

```diff
@@ -1,56 +1,63 @@
 package View.screen.GameScreenComponents;
 
 import java.util.Set;
+
 import com.badlogic.gdx.scenes.scene2d.ui.Label;
 import com.badlogic.gdx.scenes.scene2d.ui.Skin;
 import com.badlogic.gdx.scenes.scene2d.ui.Table;
 import com.badlogic.gdx.utils.Align;
+
 import java.util.HashSet;
 import Controller.GameController;
+import Model.Recruiter;
 
 /**
  * The TurnBar class represents a UI component that displays the current turn
- * and tracks
- * previous turns in a game.
- * This class is responsible for updating the turn display and managing revealed
- * turns.
+ * and tracks previous turns in a game. This class is responsible for updating
+ * the turn display and managing revealed turns.
  */
-
 public class TurnBar extends Table {
+
     private final GameController gameController;
     private String timeValue; // The current time value as a string
     private final Label timeTracker; // Label for displaying the current time
     private final Table pastTurn; // Table for displaying past turns
     private final Set<Table> revealedTurns; // Set of tables representing revealed turns
     private final Set<Integer> revealedTurnsInt; // Set of integers representing revealed turn indices
     private final Skin skin;
+    private boolean titlesAdded = false;
+    private Recruiter.RecruiterType recruiterType;
+    private final Label msType;
 
     /**
-     * Constructor for TurnBar.
-     * Initializes the UI elements and prepares the turn bar for updates.
+     * Constructor for TurnBar. Initializes the UI elements and prepares the
+     * turn bar for updates.
      *
      * @param gameController The GameController that manages game state.
      */
-
     public TurnBar(GameController gameController, Skin skin) {
-        this.setDebug(true); // Enables debug mode for visualizing layout borders
         this.gameController = gameController;
         this.skin = skin;
         // Table for displaying the turn clock
         Table turnClock = new Table();
         this.add(turnClock).expandX().fillX().top().row();
         this.timeValue = String.valueOf(gameController.getGame().getCurrentTime());
-
+        this.recruiterType = gameController.getGame().getRecruiter().getRecruiterType();
+        this.msType = new Label("Mindslip Type: " + recruiterType, skin, "half-tone");
```

Comment:
i think i was unclear in my issue earlier. The chosen mindslip should not be shown, just the possible available mindslips the recruiter may have chosen

Reply:
It's deleted now.